torchgen/gen.py

```diff
@@ -1640,8 +1640,16 @@ def get_native_function_schema_registrations(
         else:
             custom_namespace = namespace
             tab = "\t"
+            PREDEFINED_NAMESPACES = set(["quantized", "quantized_decomposed"])
```

Comment:
You can use curly here to make a set literal

Reply:
Preferably put this top level and better name eg FRAGMENT_NAMESPACES

Reply:
Also can we add it to https://github.com/pytorch/pytorch/blob/99c07735e457a2961f2319b4ba19f0d04eb47967/torchgen/model.py#L49